test/unit/driver-specs.js

```diff
@@ -18,12 +19,36 @@ describe('driver commands', () => {
     proxySpy.reset();
   });
 
-  describe('status', () => {
-    it('should send status request to WDA', async () => {
-      await driver.getStatus();
-      proxySpy.calledOnce.should.be.true;
-      proxySpy.firstCall.args[0].should.eql('/status');
-      proxySpy.firstCall.args[1].should.eql('GET');
+  describe('status caching', () => {
```

Comment:
It takes time to run this stuff on local machine. I usually test it once and then rely on Travis output while fixing it. Although this one took quite a lot of time for Travis to finish the execution :(